babel/src/main/java/org/apache/calcite/sql/babel/SqlCreateTable.java

```diff
@@ -42,19 +42,30 @@
   public final boolean isVolatile;
   public final SqlNodeList columnList;
   public final SqlNode query;
+  public final boolean withData;
+  public final OnCommitType onCommitType;
 
   private static final SqlOperator OPERATOR =
       new SqlSpecialOperator("CREATE TABLE", SqlKind.CREATE_TABLE);
 
   /** Creates a SqlCreateTable. */
   public SqlCreateTable(SqlParserPos pos, boolean replace, SetType setType, boolean isVolatile,
       boolean ifNotExists, SqlIdentifier name, SqlNodeList columnList, SqlNode query) {
+    this(pos, replace, setType, isVolatile, ifNotExists, name, columnList, query,
+        /*withData=*/false, /*onCommitType=*/OnCommitType.UNSPECIFIED);
```

Comment:
What does the /*var_name*/ do here? Is that how optional parameters are set with java?

Reply:
No, it's just a helpful hint to a reader of what the unnamed parameter is. It's typically used wherever literal parameters that have no implied context (e.g. false or 0) are passed into a function.

It's a Google style thing and probably used elsewhere as well.

Reply:
I guess I could have left off the /*onCommitType=*/ one since the enum itself contains context. But being too explicit never hurts :)

Reply:
That makes a lot of sense, thanks!